README.md

```diff
@@ -20,7 +20,7 @@ Following components are the key parts of this solution (arrows show data flow):
 1. Kubernetes Cluster: a local kubernetes cluster (K3d) running both Wasm containers and standard OCI containers as pods.
 2. Spin Shim: a Containerd shim for running Spin apps, updated the Spin shim which includes Http, Mqtt and Redis triggers.
 3. Spin Apps:
-   Spin apps are packaged as OCI container images but Spin shim extracts the Spin config and binaries from it before running the apps directly, containers are only used for packaging Spin apps, not execution.
+   Spin apps are packaged as OCI container* images but Spin shim extracts the Spin config and binaries from it before running the apps directly, containers are only used for packaging Spin apps, not execution.
```

Comment:
I think the * happened accidentally

Reply:
Explained offline: this is by design/intention, it correlates the * in diagram to signify that OCI containers for wasm are not standard containers.